util/scripts/sync_allocation_from_tas.conf

```diff
@@ -0,0 +1,9 @@
+[tas]
+url=
+client_key=
+client_secret=
+
+[portal]
+host=
+username=
+password=
```

Comment:
Please add a newline here - it is possible to configure IDEs to do this automatically for you